packages/experience/src/hooks/use-global-redirect-to.ts

```diff
@@ -1,23 +1,30 @@
 import { useCallback, useContext } from 'react';
 
 import PageContext from '@/Providers/PageContextProvider/PageContext';
+import UserInteractionContext from '@/Providers/UserInteractionContextProvider/UserInteractionContext';
 
 /**
  * This hook provides a function that process the app redirection after user successfully signs in.
  * Use window.location.replace to handle the redirection.
  * Set the global loading state to true before redirecting.
+ * Clear the user identifier user input value from session storage before redirecting.
  * This is to prevent the user from interacting with the app while the redirection is in progress.
  */
```

Comment:
Seems duplicated with the inline comments below. Since clearing storage is not the primary purpose of this method, maybe no need to mention it here? Just keep the inline comments. 

```suggestion
/**
 * This hook provides a function that processes the app redirection after the user successfully signs in.
 * Use `window.location.replace` to handle the redirection.
 * Set the global loading state to true before redirecting.
 * This is to prevent the user from interacting with the app while the redirection is in progress.
 */
```